src/plugins_3rdparty/kalign/src/KalignTask.cpp

```diff
@@ -112,6 +111,11 @@ Task::ReportResult KalignTask::report() {
     return ReportResult_Finished;
 }
 
+bool KalignTask::isAlphabetSupported(const DNAAlphabet *alphabet) {
+    return !(alphabet->getId() == BaseDNAAlphabetIds::RAW() ||
```

Comment:
Since the set of alphabet ids is the open set it is safier to check for supported alphabets ids here.